### PR TITLE
Add migration tool readme

### PR DESCRIPTION
**Pull request by junjiejiangjjj**

### CODE REVIEW

**Comment by junjiejiangjjj** [comment]
@codingjaguar

File: README.md [hunk 1]
```diff
@@ -85,6 +85,36 @@ res = client.delete(
 print(res)
 ```
 
+# migration tool
+Milvus Lite provides a command line tool that can dump milvus-lite data into a json file, which can be imported into [milvus](https://github.com/milvus-io/milvus) and [Zilliz Cloud](https://zilliz.com/cloud)(the fully managed cloud service for Milvus). The milvus-lite command  command will be installed together with milvus-lite python package
```
Comment (codingjaguar) [inline README.md:89]: command command？

File: README.md [hunk 2]
```diff
@@ -85,6 +85,36 @@ res = client.delete(
...
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -d DB_FILE, --db-file DB_FILE
+                        milvus lite db file
+  -c COLLECTION, --collection COLLECTION
+                        collection that need to be dumped
+  -p PATH, --path PATH  dump file storage dir
+```
+export data:
```
Comment (codingjaguar) [inline README.md:103]: The following example dumps all data from xxx collection that's stored in xxx (Milvus Lite database file)

File: README.md [hunk 3]
```diff
@@ -85,6 +85,36 @@ res = client.delete(
...
+```shell
+milvus-lite dump -d ./milvus_demo.db -c demo_collection -p ./data_dir
+# ./milvus_demo.db: milvus lite db file
+# demo_collection: collection that need to be dumped
+#./data_dir : dump file storage dir
+```
+Upload data to zilliz cloud:
```
Comment (codingjaguar) [inline README.md:110]: With this file, you can upload data to...

File: README.md [hunk 4]
```diff
@@ -85,6 +85,36 @@ res = client.delete(
 print(res)
 ```
 
+# migration tool
```
Comment (codingjaguar) [inline README.md:88]: All data stored in Milvus Lite can be easily exported and loaded into other types of Milvus deployment, such as Standalone on Docker, Cluster on K8s, or Zilliz Cloud (fully-managed cloud service for Milvus)

**Review by codingjaguar — APPROVED** [review]

**Pull request merged**